### ISSUE TITLE
IMOS check_conventions: 'IMOSCheck' object has no attribute '_check_value'

### ISSUE THREAD

**Issue by mhidas** [aodn/compliance-checker#30]
After merging #23 into IMOS-Checker, I now get this warning for most files checked:

```
The following exceptions occured during the imos checker (possibly indicate compliance checker issues):
imos.check_conventions: 'IMOSCheck' object has no attribute '_check_value'
```

And the check for the Conventions attribute doesn't seem to to happen. I guess the merge didn't work perfectly.

**Comment by fxmzb123** [comment]
@mhidas the check conventions has been added, and the cause for this is because this fix is happened before the tide up branch and the help method _check_value has been moved to Util module. so I think I need to rebase other branches before we merge. This has been fixed in a new branch and I created a new pull request.

**Comment by mhidas** [comment]
Fixed via #31
